geoips/plugins/modules/procflows/single_source.py

```diff
@@ -1860,13 +1863,23 @@ def call(fnames, command_line_args=None):
 
         for output_product in final_products:
             output_checker = output_checkers.get_plugin(output_product)
-            retval += output_checker(
-                output_checker,
-                compare_path.replace("<product>", product_name)
-                .replace("<procflow>", "single_source")
-                .replace("<output>", output_formatter),
-                [output_product],
-            )
+            if output_checker.name == "image":
+                retval += output_checker(
+                    output_checker,
+                    compare_path.replace("<product>", product_name)
+                    .replace("<procflow>", "single_source")
+                    .replace("<output>", output_formatter),
+                    [output_product],
+                    image_compare_threshold,
+                )
+            else:
+                retval += output_checker(
+                    output_checker,
+                    compare_path.replace("<product>", product_name)
+                    .replace("<procflow>", "single_source")
+                    .replace("<output>", output_formatter),
+                    [output_product],
+                )
```

Comment:
Consider updating to "output_checker_kwargs", and passing in to all output_checker plugins